src/main/java/com/fasterxml/jackson/module/jsonSchema/types/ArraySchema.java

```diff
@@ -133,6 +142,15 @@ protected boolean _equals(ArraySchema that)
                  && equals(getUniqueItems(), that.getUniqueItems())
                  && super._equals(that);
      }
+
+	@Override
+	public String toString() {
+		try {
+			return new ObjectMapper().writeValueAsString(this);
```

Comment:
Constructing a new `ObjectMapper` from within `toString()` is not something to do, for multiple reasons (extra slow performance, non-configurability, for example).
As a practical consequence, none of Jackson components serialize full valid JSON (or, rather, is guaranteed to do so) from `toString()`, and users should not assume results are useful for anything other than debugging.